src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -1014,6 +1036,7 @@ static void write_action(void* gt, grpc_error* error) {
                         grpc_combiner_scheduler(t->combiner)));
 }
 
+/* Callback from the grpc_endpoint after bytes have been written on the wire */
```

Comment:
saying 'on the wire' is wrong

Reply:
what should this be instead? - should we say after calling `sendmsg` ?